ibis/backends/sql/compiler.py

```diff
@@ -94,7 +92,9 @@ def __init__(self, namespace: str | None = None) -> None:
 
     def __getattr__(self, name: str) -> Callable[..., sge.Func]:
         name = ".".join(filter(None, (self.namespace, name)))
-        return lambda *args, **kwargs: sg.func(name, *map(sge.convert, args), **kwargs)
+        return lambda *args, **kwargs: sg.func(
+            name, *map(sge.convert, args), **kwargs, copy=False
```

Comment:
Passing `copy=False` here speeds up the slow expression from the bug report from `15s` to `6.5s` but breaks a couple of mssql tests.

Reply:
I pushed up a commit that allows selectively copying arguments passed to `func` and enables it for the `mssql` backend.